driver/resthandler.go

```diff
@@ -0,0 +1,364 @@
+//
+// Copyright (c) 2019 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package driver
+
+import (
+	"context"
+	"fmt"
+	sdk "github.com/edgexfoundry/device-sdk-go"
+	"github.com/edgexfoundry/device-sdk-go/pkg/models"
+	"github.com/edgexfoundry/go-mod-core-contracts/clients"
+	"github.com/edgexfoundry/go-mod-core-contracts/clients/logger"
+	"github.com/gorilla/mux"
+	"github.com/spf13/cast"
+	"io/ioutil"
+	"math"
+	"net/http"
+	"time"
+)
+
+const (
+	deviceNameKey     = "deviceName"
+	resourceNameKey   = "resourceName"
+	apiResourceRoute  = clients.ApiDeviceRoute + "/{" + deviceNameKey + "}/{" + resourceNameKey + "}"
```

Comment:
This is the same route as is used by other device services for their normal get and set operations. We're handling POST operations, so it doesn't conflict, but it could be a source of confusion.

Reply:
Right, but also /api/v1/device/{deviceId}/{command}

Reply:
Ahh, yes your are correct. I missed that one.

How about I use `/api/v1/resource/{deviceName}/{resourceName}` since resource value is what we are POSTing?

Reply:
No, looks fine to me